packages/ses/src/error/note-log-args.js

```diff
@@ -89,26 +84,35 @@ const spliceOut = cell => {
  * rather than a `WeakMapStore`-like interface. To work before `lockdown`,
  * the implementation must use `freeze` manually, but still exhausively.
  *
- * It does not hold onto anything weakly. The only sense in which it is
- * WeakMap-like is that it does not support enumeration.
+ * It implements the WeakMap interface, and holds its keys and values only
```

Comment:
I think "and values" is erroneous ? A WeakMap only holds the key weakly, and simulates a strong link between the key and the value.